languages/julia/exercises/concept/poetry-club/runtests.jl

```diff
@@ -0,0 +1,151 @@
+using Test
```

Comment:
Obviously this is not how you'd write a test file for this. But to keep it understandable without prereqs and to avoid giving away the solution, all strings are written out.

Reply:
I would add this as comments! It's a good comment to have.

Reply:
It's a comment in the test generation file, not the file exposed to students. I added the GH comment so that it stands out more for reviews